proxy/http2/Http2ClientSession.cc

```diff
@@ -332,6 +333,19 @@ Http2ClientSession::set_half_close_local_flag(bool flag)
   half_close_local = flag;
 }
 
+int64_t
+Http2ClientSession::xmit(const Http2TxFrame *frame)
```

Comment:
Do we want to pass a pointer? Should we check nullptr?

Reply:
Will change to const reference.